gstorm/include/detail/algorithms/reduce.h

```diff
@@ -36,9 +36,9 @@ auto reduce(InRng &in, T init, BinaryFunc func,
 
       auto start = gid * wpt;
 
-      value_type sum = value_type();
+      T sum = *(inIt + start);
 
-      for (size_t i = 0; i < wpt; ++i)
+      for (size_t i = 1; i < wpt; ++i)
```

Comment:
Could you explain why this is now starting from 1 ? A comment in the code would help